### PR TITLE
Anonymise name

### PR DESCRIPTION
**Pull request by natashamcintyre**

### CODE REVIEW

File: test/fixtures/vcr_cassettes/get_users.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
...
...: \"https://secure.gravatar.com/avatar/e0e3d90e70d3e578956e833528d6e487\", \"id\": \"user_tGK8Xp5Fl6LpcFfAWXWdKkFg2PzgVWEv8I96texckYz\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-09-18T10:25:41.547000+00:00\", \"email\": \"bob@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Andy\", \"last_name\": \"Harris\", \"date_updated\": \"2017-12-18T03:15:04.656000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/f0638325de8165e27b474d9f9a119634\", \"id\": \"user_v9yUukdKu0f6dUE0lorzRWrkwMebLBZbjpvOzkKtNma\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-08-29T13:48:36.306000+00:00\", \"email\": \"andy@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Bonnie\", \"last_name\": \"Pooley\", \"date_updated\": \"2017-12-18T03:16:19.179000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/9bdce6c2f30296f80edf18b80c642d51\", \"id\": \"user_xdG0HLQezJDX2ALlcFevERSCv2PeEc3tbipa308oVnn\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-12-13T15:30:52.002000+00:00\", \"email\": \"bonnie@nested.com\"}]}",
```
Comment (natashamcintyre) [inline test/fixtures/vcr_cassettes/get_users.json:16]: Not a nice diff. I've changed him to "Mister Blobby"

Comment (jevdokd) [reply test/fixtures/vcr_cassettes/get_users.json:16]: image

**Review by jevdokd — APPROVED** [review]
LGTM!

**Comment by jevdokd** [comment]
oh forgot to ask - can you please add a description why we're doing this. I know it's in the ticket, but the git history will stay forever (unless we delete it), but I can't say the same about Shortcut.

**Pull request merged**